PassiFy/PasswordGenerator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
Moved the password generator methods into their own class so that we could separate the logic out. This helps keep your project cleaner, and you can tell at a glance what something does.